lib/jellyfishService.js

```diff
@@ -260,10 +263,15 @@ function createServer(serverConfig, mongoClient, seagullClient, userApiClient, g
           }
         ],
         function(err) {
-          if (updatesSummary == true) {
-            streamDAO.setSummaryOutdated(storedUserId, function(err){
+          if (updatesSummary.cgm == true) {
+            streamDAO.setSummaryOutdated(storedUserId, 'cgm', function(err){
+              if (err != null) {
+                log.warn(err, 'Problem marking cgm summary outdated for user[%s].', userid);
+              }
+            });
+            streamDAO.setSummaryOutdated(storedUserId, 'bgm', function(err){
```

Comment:
Should you check if `updatesSummary.bgm === true` here?

Reply:
This entire code block should be tested too

Reply:
fixed, but did not add testing for this block. I tried to find a way to test it, but it would require... more fixing than its probably worth. I am open to suggestions as this code is impactful if it breaks, but unit testing it looks very painful.

Reply:
Why painful? We already have tests in `ingestionApiTest.js` - can't those be extended to check for lack or presence of summaries?